src/loaders/lottie/tvgLottieModel.h

```diff
@@ -180,6 +180,19 @@ struct LottieFont
     Origin origin = Embedded;
 };
 
+struct LottieMarker
+{
+    enum SelectedSegment:std::uint8_t { NONE = 0, MARKER, RANGE };
```

Comment:
`enum SelectedSegment : uint8_t { NONE = 0, MARKER, RANGE };`

(Plus) I think it doesn't have state (whether it's selected or not) it self in this context. So the prefix `Selected` could be ambigious.